Cargo.toml

```diff
@@ -79,9 +79,9 @@ syn = "2.0.38"
 tame-oauth = "0.10.0"
 tempfile = "3.1.0"
 thiserror = "1.0.29"
-tokio = "1.14.0"
+tokio = "1.38.0"
 tokio-test = "0.4.0"
-tokio-tungstenite = "0.21.0"
+tokio-tungstenite = "0.23.0"
```

Comment:
afaikt this tokio bump to 1.38 is unnecessary. it only depends on 1
https://crates.io/crates/tokio-tungstenite/0.23.0/dependencies / https://github.com/snapview/tokio-tungstenite/blob/master/Cargo.toml#L33

bumping tungstenite by itself is worth it though because that's likely what is leading to version conflicts

Reply:
It also is dependant on crate.io somehow. I cannot force tokio 1.14 into my project once 1.38 has been added. Deleting lockfile and target dir does not help. However, We should not have a useless crate dependency in there that results in larger binary size, confuses cargo and leads to strange errors.

Reply:
The `1.14` is mostly informational (last known feature we took from tokio came in this version) and imposes no real constraints other than >=1.14. It does not mean that you will be able to build a maximal kube + the rest of the ecosystem with 1.14, cargo should pick one satisfying the union of all constraints, if you try to force 1.14 that might indeed fail.

Putting 1.38 on our end means we force every user to also upgrade tokio with the next kube release which i am not a fan of.

Reply:
Seems I can't reproduce the dependency error anymore. So I guess it was time dependant? Maybe tokio just made a release?

Anyway updating to your suggestion.

Reply:
Updating `tokio-tungstenite` is welcome regardless. Thank you.